bootstrap.php

```diff
@@ -189,6 +189,20 @@ function () {
 	}
 }
 
+require BLUEHOST_PLUGIN_DIR . '/inc/LoginRedirect.php';
+LoginRedirect::init();
+
+// Disable Yoast SEO onboarding redirect
+add_action(
```

Comment:
What do you think about moving this into the LoginRedirect class? It would help keep the bootstrap file cleaner?

Not a blocker at all, but thought I'd bring it up.

Reply:
@circlecube Excellent idea, I moved it here: https://github.com/bluehost/bluehost-wordpress-plugin/pull/505/commits/b04cd2bd9684a6ff4d39ace8bc4b1eb879930bea